kafka_utils/kafka_cluster_manager/cmds/command.py

```diff
@@ -157,9 +157,10 @@ def process_assignment(self, assignment, allow_rf_change=False):
     def get_reduced_assignment(
         self,
         original_assignment,
-        new_assignment,
+        cluster_topology,
         max_partition_movements,
         max_leader_only_changes,
+        max_movement_size=float('inf'),
```

Comment:
Minor: Should we just import and use DEFAULT_MAX_MOVEMENT_SIZE here for consistency?